.circleci/config.yml

```diff
@@ -55,6 +55,9 @@ jobs:
       - when:
           condition: <<parameters.push>>
           steps:
+            - run:
+                name: DockeHub login
+                command: docker login --username "$DOCKER_LOGIN" --password "$DOCKER_PASSWORD"
```

Comment:
would recommend sticking with the STDIN approach instead, that way the password doesn't end up in the process listing?